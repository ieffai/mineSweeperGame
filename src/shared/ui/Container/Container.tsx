import { Container as MuiContainer, type ContainerProps as MuiContainerProps } from '@mui/material';

export { MuiContainer as Container };
export type { MuiContainerProps as ContainerProps };
