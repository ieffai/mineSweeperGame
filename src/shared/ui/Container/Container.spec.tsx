import { render } from '@testing-library/react';
import { Container } from './Container';

describe('Container', () => {
  it('should render successfully', () => {
    const { asFragment } = render(<Container />);
    expect(asFragment()).toMatchSnapshot();
  });
});
