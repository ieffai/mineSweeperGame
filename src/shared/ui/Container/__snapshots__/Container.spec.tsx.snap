// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Container > should render successfully 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-5c1adp-MuiContainer-root"
  />
</DocumentFragment>
`;
